jobs/quicksuggest2bq/tests/test_main.py

```diff
@@ -57,7 +64,7 @@ def test_suggestion_properties_are_properly_parsed(self):
         KintoSuggestion(**SAMPLE_SUGGESTION)
 
     def test_suggestion_download(self, mocked_kinto_client):
-        suggestions = download_suggestions(mocked_kinto_client)
-        assert len(suggestions) == 1
-        assert SAMPLE_SUGGESTION["id"] in suggestions
-        assert suggestions[SAMPLE_SUGGESTION["id"]].title == SAMPLE_SUGGESTION["title"]
+        suggestions = list(download_suggestions(mocked_kinto_client))
+        assert len(suggestions) == 2
```

Comment:
Why is this 2 and not 3?

Reply:
I added an icon record to `mock_records`, wanted to make sure it won't be downloaded here. I've split it out into its own test to avoid confusion.

Reply:
Thanks for clarifying!